### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_ColorContrast: The checkbox color contrast is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10620]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_AzureDataStudioStayHealthy;#A11ySev3;#A11yTCS;#A11yWCAG2.1;#NonBenchmark#DesktopApp;#MAS1.4.11;#SQL Azure Data Studio;#FTP;#ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen.
4 .Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
The  color contrast of the check box "show welcome page on start up" when not checked  is at 2.119:1 which is below 3:1

**Expected:**
The color contrast of the check box should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in identifying the check box,.

**Attachment link for Reference:**
![A11y_AzureDataStudioStayHealthy_Welcome_Resources_ColorContrast](https://user-images.githubusercontent.com/24580570/83240225-b5891080-a1b6-11ea-8630-52eef07729cc.png)

**Comment by alanrenmsft** [comment]
@fsteffi this is an upstream issue (chromium), vscode also has the same problem, see screenshot below

<img width="1466" alt="Screen Shot 2020-06-18 at 11 23 19 AM" src="https://user-images.githubusercontent.com/13777222/85057978-4c157580-b156-11ea-9960-e5b991816dc5.png">

please open an issue for vscode.
